### PR TITLE
German Translation for v0.12.1.0

### PR DESCRIPTION
**Pull request by Rafficer**
Updated all strings for the point release.

Message to reviewers: Everything that's set to `type="unfinished"` currently is intentional as in my opinion the original string should be used for that.

### CODE REVIEW

**Comment by erciccione** [comment]
ref #1377

File: translations/monero-core_de.ts [hunk 1]
```diff
@@ -1238,138 +1238,138 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
...
         <location filename="../pages/SharedRingDB.qml" line="364"/>
         <source>I intend to spend on key-reusing fork(s)</source>
-        <translation type="unfinished"></translation>
+        <translation>I habe vor auf einem Key wiederverwendenden Monero-Klon Ausgaben zu tätigen</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="378"/>
         <source>I might want to spend on key-reusing fork(s)</source>
-        <translation type="unfinished"></translation>
+        <translation>I möchte vielleicht an einem Key wiederverwendenden Monero-Klon Ausgaben tätigen</translation>
```
Comment (Flenst) [inline translations/monero-core_de.ts]: "Ich möchte..."

File: translations/monero-core_de.ts [hunk 2]
```diff
@@ -1685,32 +1685,32 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
...
         <location filename="../pages/Transfer.qml" line="227"/>
         <source>Default</source>
-        <translation type="unfinished">Standard</translation>
+        <translation>Standard</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="229"/>
         <source>Normal (x1 fee)</source>
-        <translation type="unfinished"></translation>
+        <translation>Normal (1x Gebühr)</translation>
```
Comment (Flenst) [inline translations/monero-core_de.ts]: x1 besser als 1x denke ich

Comment (Rafficer) [reply translations/monero-core_de.ts]: `1-fache`, weil bei den anderen Gebuehren genauso. Siehe Zeile 1508.

**Comment by Rafficer** [comment]
Fixed everything according to @Flenst's review

File: translations/monero-core_de.ts [hunk 3]
```diff
@@ -784,97 +784,97 @@
...
         <source>Help</source>
-        <translation type="unfinished"></translation>
+        <translation>Hilfe</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="206"/>
         <location filename="../pages/Receive.qml" line="435"/>
         <source>&lt;p&gt;This QR code includes the address you selected above andthe amount you entered below. Share it with others (right-click-&gt;Save) so they can more easily send you exact amounts.&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;Dieser QR-Code enthält die oben gewählte Adresse und den Bentrag der unten eingegeben wurde. Teile ihn mit anderen (Rechtsklick-&gt;Speichern), dass diese dir einfacher exakte Beträge senden können.&lt;/p&gt;</translation>
```
[the remark this answers is not in the document]
Comment (Rafficer) [reply translations/monero-core_de.ts]: Hab da den Satz jetzt auch noch minimal umgebaut.

File: translations/monero-core_de.ts [hunk 4]
```diff
@@ -1238,138 +1238,138 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
...
         <location filename="../pages/SharedRingDB.qml" line="346"/>
         <source>Set Ring</source>
-        <translation type="unfinished"></translation>
+        <translation>Ring setzen</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="364"/>
         <source>I intend to spend on key-reusing fork(s)</source>
-        <translation type="unfinished"></translation>
+        <translation>I habe vor auf einem Key wiederverwendenden Monero-Klon Ausgaben zu tätigen</translation>
```
Comment (MaxXor) [inline translations/monero-core_de.ts]: `I` -> `Ich`

Comment (Rafficer) [reply translations/monero-core_de.ts]: Already fixed in 52d6ea08a0e15e1aa89351b832df317e7d59c12c

Comment (Rafficer) [reply translations/monero-core_de.ts]: Okay, nvm, it was another string. Will fix that, sorry.

Comment (MaxXor) [reply translations/monero-core_de.ts]: Please squash the commits into one, when you are done.

**Comment by Rafficer** [comment]
Fixed, squashed and pushed. Thanks for the review @MaxXor!

File: translations/monero-core_de.ts [hunk 5]
```diff
@@ -1038,12 +1038,12 @@
...
         <location filename="../pages/Settings.qml" line="680"/>
         <source> &lt;a href=&apos;#&apos;&gt; (Click to change)&lt;/a&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation> &lt;a href=&apos;#&apos;&gt; (Zum Ändern drücken)&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="682"/>
         <source>Set a new restore height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Setze eine neue Wiederherstellungshöhe</translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: missing `:`

File: translations/monero-core_de.ts [hunk 6]
```diff
@@ -1238,138 +1238,138 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
...
         <location filename="../pages/SharedRingDB.qml" line="189"/>
         <source>Load</source>
-        <translation type="unfinished"></translation>
+        <translation>Laden</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="205"/>
         <source>Or manually blackball/unblackball a single output:</source>
-        <translation type="unfinished"></translation>
+        <translation>Oder manuell einen einzelnen Output blackballen/entblackballen</translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: missing `:`

File: translations/monero-core_de.ts [hunk 7]
```diff
@@ -1238,138 +1238,138 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
...
         <location filename="../pages/SharedRingDB.qml" line="392"/>
         <source>Relative</source>
-        <translation type="unfinished"></translation>
+        <translation>Relativ</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="408"/>
         <source>Segregation height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Segregationshöhe</translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: missing `:`

File: translations/monero-core_de.ts [hunk 8]
```diff
@@ -1685,32 +1685,32 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
...
         <location filename="../pages/Transfer.qml" line="82"/>
         <source>Ring size: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Ringgröße: %1</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="156"/>
         <source>This page lets you sign/verify a message (or file contents) with your address.</source>
-        <translation type="unfinished"></translation>
+        <translation>Diese Seite lässt dich Nachrichten (oder Dateiinhalte) mit deiner Adresse signieren/verifizieren </translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: whitespace at the end of the sentence instead of `.`

**Comment by Rafficer** [comment]
Yeah, I can't find a translation that sounds good... @Flenst do you maybe have a suggestion?

Will correct the rest.

**Comment by MaxXor** [comment]
@Rafficer Maybe `ausgabefähiges Wallet` or just leave it as it is. Just like `View-only Wallet`

**Comment by Flenst** [comment]
@Rafficer no idea. Don't even get where it makes sense, you do not spend wallets (do you?), but Moneroj. Don't know.

**Comment by Rafficer** [comment]
Well it's the opposite of a view only wallet with just the view key :P

**Comment by Rafficer** [comment]
It's used for something specific so I would leave it just as spendable wallet.

**Comment by Rafficer** [comment]
Everything should be fixed now.

**Comment by Rafficer** [comment]
Wrong button...

**Pull request merged**